FIAT/P0.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, ref_el):
                 entity_ids[dim][entity] = [entity] if dim == sd else []
                 entity_permutations[dim][entity] = perms
 
-        super(P0Dual, self).__init__(nodes, ref_el, entity_ids, entity_permutations)
```

Comment:
Why the pervasive `super(foo, self)` to `super()` conversion?

Reply:
It's just better style